app/components/jobs/workflow-card.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  workflow: null,
+  onPicked: null,
```

Comment:
I'm not following you here. workflow should be passed a workflow model not a function.
Also, why would you need a type hint for a named attribute vs a positional parameter?

Reply:
The comment was for `onPicked`, not workflow, . When someone uses this component, they can't do it without providing a value for `onPicked`. If they don't the component tries to call `null` as a function.

So, I suggest putting in a default function implementation for `onPicked`, mostly as a hint to the user on what the type of this property should be, since it's otherwise hard to determine.

The named attribute vs positional parameter doesn't make any technical difference here, just that it's not obvious to someone using this code that they have to provide values for named attributes when using the component. Declaring positional parameters encourages providing values when using the component.